app-editors/bluefish/bluefish-2.2.12.recipe

```diff
@@ -0,0 +1,162 @@
+SUMMARY="A powerful HTML editor for experienced web designers and programmers"
+DESCRIPTION="Long ProjectX description.
+Spanning several paragraphs, maybe
+ - containing
+ - bullet points etc.
+You can also have really long lines that do not fit in 80 characters and use \
+a backslash to wrap the line."
+HOMEPAGE="http://bluefish.openoffice.nl/"
+COPYRIGHT="2014 Developer name
+	2016 Another developer's name"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://www.bennewitz.com/bluefish/stable/source/bluefish-$portVersion.tar.gz"
```

Comment:
```suggestion
SOURCE_URI="https://www.bennewitz.com/bluefish/stable/source/bluefish-$portVersion.tar.gz"
```

Reply:
Hence wip label, just wanted to push what I had right now, needs fine-tuning :)